### PR TITLE
made changes in create receipt

### PR DESCRIPTION
**Pull request by PunamSonawane**
Made some changes in createReceiptVision handler and createReceipt handler.

### CODE REVIEW

File: server/api/createReceipt_handler.py [hunk 1]
```diff
@@ -24,33 +21,24 @@ def createReceipt():
...
-        # Storing the data in aws s3 buckets
-        s3=boto3.resource('s3',aws_access_key_id=ACCESS_KEY_ID,aws_secret_access_key=ACCESS_SECRET_KEY,config=Config(signature_version='s3v4'))
-        location=s3.Bucket(BUCKET_NAME).put_object(Key=p,Body=pict_file)
-        # Url where we store our file
-        picture_url = 'https://%s.s3.ca-central-1.amazonaws.com/%s' % (BUCKET_NAME, p)
+        title=request.get_json()['title']
+        amount=request.get_json()['amount']
+        category=request.get_json()['category']
+        receipt_date=request.get_json()['receipt_date']
+        create_date=request.get_json()['create_date']
```
Comment (HitikshaPanchal) [inline server/api/createReceipt_handler.py]:
```suggestion
        create_date=request.get_json()['date_created']
```

File: server/api/createReceiptVision_handler.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                for j in range(len(new_lst)):
+                    digit="{}".format(new_lst[j].description)
+                    
+                    dig=re.findall('\d*\.?\d+',digit)
+                    
+                    if len(dig)!=0:
+                        amount=dig[0]
+                        break
+                    else:
+                        pass
```
Comment (bonnieli) [inline server/api/createReceiptVision_handler.py]: delete

File: server/api/createReceiptVision_handler.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                break
+        if amount==None:
+            amount=0   
+        else: 
+            pass
```
Comment (bonnieli) [inline server/api/createReceiptVision_handler.py]: delete these passes since they're not doing anything

File: server/api/email_handler.py [hunk 4]
```diff
@@ -50,9 +50,8 @@ def sendEmail():
         from_email=SENDER_EMAIL,
         to_emails=user_email,
         subject='Report from Receipt Tracker',
-        html_content='<p>Dear User,<br/><br/>This email has attached report for the requested month and year.<br/>We really appreciate your business and always looking for your feedback to improve our service.<br/><br/>Have a great day!</p>'
+        html_content='<p>Dear User,<br/><br/>This email has attached report for the requested month and year.<br/>We really appreciate your bussiness and always looking for your feedback to improove our service.<br/><br/>Have a great day!</p>'
```
Comment (bonnieli) [inline server/api/email_handler.py:53]: you may have to update your branch to the latest code

File: server/api/linechart_handler.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+from flask import jsonify, Blueprint
+from flask import Flask, request, json , make_response
+from database.models import user,receipt,picture
+import datetime
+import pandas as pd
+
+linechart_handler = Blueprint('linechart_handler',__name__)
+
+# Create a route for '/getReceipts/<duration>'
+@linechart_handler.route('/linechart',methods=['GET'])
```
Comment (bonnieli) [inline server/api/linechart_handler.py:11]:
```suggestion
@linechart_handler.route('/line_chart',methods=['GET'])
```

**Review by bonnieli — COMMENTED** [review]
next time, you may benefit to split out your features into different branches because i noticed this PR is for 3 different features: Google CV, Line chart, and change to create receipt

in addition, you may also want to take at installing flake8 as a linter for better readability of code
https://medium.com/python-pandemonium/what-is-flake8-and-why-we-should-use-it-b89bd78073f2

**Pull request merged**